### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alai97**
I updated the instructions in the take home writing exercise to be more clear and consistent.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,78 +1,81 @@
...
+* Create a pull request and submit the PR link in Greenhouse.
 
-Don’t forget to read the [References](https://github.com/jeremy-lq/hiring-engineers/blob/tech-writer/README.md#references)
+If you have any questions, create an issue in this repository and contact Kaylyn Sigler.
 
-## Prerequisites - Setup the environment
+## Prerequisites
 
-You can utilize [any OS/host](https://app.datadoghq.com/account/settings#agent) to complete this exercise. However, we recommend one of the following approaches:
+You can utilize [any OS/host](https://app.datadoghq.com/account/settings#agent) to complete this exercise. To setup your environment, use one of the following approaches:
```
Comment (cswatt) [inline README.md:19]: this is too prescriptive. we don't actually require that people set up a VM/container, they can use any approach.

File: README.md [hunk 2]
```diff
@@ -1,78 +1,81 @@
...
-To submit your answers:
-
+* Read the [prerequisites](https://github.com/jeremy-lq/hiring-engineers/blob/tech-writer/README.md#prerequisites) and [references](https://github.com/jeremy-lq/hiring-engineers/blob/tech-writer/README.md#references) before the exercises.
 * Fork this repo.
-* Answer the questions in answers.md
-* Commit as much code and images as you need to support your answers.
-* Submit a pull request.
-* Don't forget to include links to your dashboard(s), or, even better, links and screenshots. We recommend that you include your screenshots inline with your answers.
+* Complete your responses in the `answers.md` file of your forked repo.
+* Add links to your dashboard and screenshots inline with your responses when prompted.
```
Comment (cswatt) [inline README.md:10]: too prescriptive—we want people to make this decision themselves

File: README.md [hunk 3]
```diff
@@ -1,78 +1,81 @@
...
+* Commit as much code as you need to support your responses.
+* Proofread your responses.
```
Comment (cswatt) [inline README.md:12]: leave this out. people should decide to proofread on their own without being prompted

File: README.md [hunk 4]
```diff
@@ -1,78 +1,81 @@
...
 
-## Final Question:
+**Bonus Question**: What does the Anomaly graph display?
+
+## Final Exercise: Blog Post
 
-The Datadog community has written a substantial number of high-quality integrations and libraries. Select one from [this page](https://docs.datadoghq.com/developers/libraries/). With this selection in mind, write a blog post that announces your selection and explains the benefits it offers our users/community. The post should show that you have a clear and in-depth understanding of the technology by diving into installation, configuration, usage, and best practices along with code samples where applicable. You should also thank the contributor for their effort.
+The Datadog community has written a substantial number of high-quality integrations and libraries on [this page](https://docs.datadoghq.com/developers/libraries/). 
+
+Write a blog post that announces your topic and explains the benefits it offers our users/community. Thank the contributor for their efforts and include images, code samples, and diagrams where applicable. You can search [The Monitor](https://www.datadoghq.com/blog/) for past posts.
```
Comment (cswatt) [inline README.md:56]:
```suggestion
Write a blog post that announces your topic and explains the benefits it offers our users/community. Thank the contributor for their efforts and include images, code samples, and diagrams where applicable.
```

don't give too many hints lol

File: README.md [hunk 5]
```diff
@@ -1,78 +1,81 @@
...
 * Take a snapshot of this graph and use the @ notation to send it to yourself.
-* **Bonus Question**: What is the Anomaly graph displaying?
 
-## Final Question:
+**Bonus Question**: What does the Anomaly graph display?
+
+## Final Exercise: Blog Post
 
-The Datadog community has written a substantial number of high-quality integrations and libraries. Select one from [this page](https://docs.datadoghq.com/developers/libraries/). With this selection in mind, write a blog post that announces your selection and explains the benefits it offers our users/community. The post should show that you have a clear and in-depth understanding of the technology by diving into installation, configuration, usage, and best practices along with code samples where applicable. You should also thank the contributor for their effort.
+The Datadog community has written a substantial number of high-quality integrations and libraries on [this page](https://docs.datadoghq.com/developers/libraries/).
```
Comment (cswatt) [inline README.md:54]:
```suggestion
The Datadog community has written a substantial number of [high-quality integrations and libraries](https://docs.datadoghq.com/developers/libraries/). Select one as a topic.
```

**Review by cswatt — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (alai97) [reply README.md:10]: For both bullet points, or just the second one about adding links/screenshots?

**Pull request closed without merging**